contracts/base/LendingPool.sol

```diff
@@ -23,27 +23,27 @@ import "../util/ZeroCollateralCommon.sol";
 
 // Interfaces
 import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
-import "../interfaces/DAIPoolInterface.sol";
-import "../interfaces/LenderInfoInterface.sol";
+import "../interfaces/LendingPoolInterface.sol";
+import "../interfaces/LendersInterface.sol";
 import "../interfaces/ZDAIInterface.sol";
 
 // Contracts
 import "./Initializable.sol";
 
 
 /**
-    @notice The DAIPool contract holds all of the DAI that lenders transfer into the protocol. It is the contract that lenders interact with to deposit and withdraw DAI including interest. The DAIPool interacts with the LenderInformation contract to ensure DAI balances and interest owed is kept up to date.
+    @notice The LendingPool contract holds all of the tokens that lenders transfer into the protocol. It is the contract that lenders interact with to deposit and withdraw their tokens including interest. The LendingPool interacts with the LenderInformation contract to ensure token balances and interest owed is kept up to date.
  */
-contract DAIPool is DAIPoolInterface, Initializable {
+contract LendingPool is LendingPoolInterface, Initializable {
     using AddressLib for address;
 
     /* State Variables */
 
-    IERC20 public dai;
+    IERC20 public token;
 
     ZDAIInterface public zdai;
```

Comment:
If we are now calling it the `lendingPool` not the `daiPool` should `zDAI` be renamed too?

Reply:
Good catch!